### PR TITLE
adding logic for displaying apply cards

### PR DESCRIPTION
**Pull request by s-laugh**
### Description

List of proposed changes:

- Adding the logic for displaying apply/advertisement cards
-

### CODE REVIEW

**Review by amirpourpanahi — APPROVED** [review]

**Comment by s-laugh** [comment]
Waiting for a second review with all the changes in.

**Review by amirpourpanahi — APPROVED** [review]

**Pull request merged**
